tests/zero_code_change/tensorflow2_integration_tests.py

```diff
@@ -77,6 +80,103 @@ def test_keras_v2(script_mode: bool = False, eager_mode: bool = True):
         trial = smd.create_trial(path=sim.out_dir)
         assert len(trial.steps()) > 0, "Nothing saved at any step."
         assert len(trial.tensor_names()) > 0, "Tensors were not saved."
+        assert len(trial.tensor_names(collection="losses")) > 0
+
+
+def helper_test_keras_v2_json_config(
+    json_file_contents, script_mode: bool = False, eager_mode: bool = True
+):
+    """ Works as intended. """
```

Comment:
Can you improve this comment to describe what is intended?

Reply:
done in the latest commit